test/node/image.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
 
 const fs = require('fs');
+const path = require('path');
+const {Buffer} = require('buffer');
```

Comment:
The linter didn't complain that it should be `const { Buffer }` vs `const {Buffer`? Odd!